tutorials/seqcopy/README.md

```diff
@@ -0,0 +1,140 @@
+# Sequence copy model
+This tutorial will show you the basic usage of Sockeye on a task simpler than Machine Translation: copying a sequence.
+We will generate sequences consisting of digits of variable lengths.
+The task is then to train a model that copies the sequence from the source to the target.
+This task is on the one hand difficult enough to be interesting and on the other and allows for quickly training a model.
+
+## Setup
+For this tutorial we assume that you have successfully [installed](../../README.md#installation) Sockeye.
+Other than that there are no additional dependencies.
+We will be using scripts from the Sockeye repository, so you should either clone the repository or manually download 
+the scripts.
+Just as a reminder: Everything is run using Python 3, so depending on your setup you may have to replace `python` with
+`python3` below.
+All of the commands below assume you're running on a CPU.
+If you have a GPU available you can simply remove `--use-cpu`.
+
+## 1. Generating the data
+As a first step we will generate a synthetic data set consisting of random sequences of digits.
+These sequences are then split into disjoint training and development sets.
+Run the following command to create the data set:
+
+```bash
+python genseqcopy.py
+```
+
+After running this script you have a training (`train.source`, `train.target`) and a development data set
+(`dev.source`, `dev.target`). The sequences that were generate look like this:
+
+```
+2 3 5 5 4 6 7 0 3 8 10 9 3 6
+9 9 1 5 3 0 5 4 0 8 8 5 7 7 8 7 3 1 0
+9 1 9 7 9 1 9 9 9 3 9 3 2 8 0 1 6 10 4 3 1 9 2 7 1 5 7 7 5 5
+2 1 4 10 7 7 7 2 10 9 4 9 9 7 8 4 10 6 8 2 6 7 5 3 2
+4 6 0 7 8 8 6 3 4 10 2 10 6 9 5 3
+8 0 5 4 1 8 0 8 7 4 4 0 0 9 5 8 9
+```
+
+## 2. Training
+
+Now that we have some training data to play with we can train our model. Start training by running
+the following command:
+
+```bash
+python3 -m sockeye.train -s train.source \
+                         -t train.target \
+                         -vs dev.source \
+                         -vt dev.target \
+                         --num-embed 32 \
+                         --rnn-num-hidden 64 \
+                         --attention-type dot \
+                         --use-cpu \
+                         --metrics perplexity accuracy \
+                         --max-num-checkpoint-not-improved 3 \
+                         -o seqcopy_model
+```
+
+This will train a 1-layer RNN model with a bidirectional LSTM as the encoder side and a uni-directional LSTM
```

Comment:
"as the encoder" or "on the encoder side"

Reply:
changed